dev-docs/bidders/addigi.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: bidder
+title: Addigi
+description: Addigi Bidder Adapter
+biddercode: addigi
+aliasCode : smarthub
+usp_supported: true
+coppa_supported: true
+schain_supported: true
+dchain_supported: true
+media_types: banner, video, native
+safeframes_ok: true
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+pbjs: true
+pbs: true
```

Comment:
```suggestion
pbs: false
```

If you set this to false, we can merge this and you can open a second PR that links to the prebid-server PR. Usually prebid server takes a bit longer to merge and release

Reply:
ok I changed pbs to false